infrastructure/terraform/shared/vpcs.tf

```diff
@@ -116,4 +120,28 @@ module "datascience_vpc" {
   providers = {
     aws = "aws.datascience"
   }
+}
+
+# Used by:
+# - Item requesting service
+# TODO: Move all catalogue services into this VPC
+
+module "catalogue_vpc" {
+  source = "github.com/wellcometrust/terraform//network/prebuilt/vpc/public-private-igw?ref=v19.5.3"
+
+  name = "catalogue-172-18-0-0-16"
```

Comment:
tiny nit: not in love with this name; if the CIDR block changes suddenly the name no longer matches. Why not call it catalogue?

Reply:
The CIDR block is fixed. Our VPCs subnets are all intended to be non-intersecting and need active planning to be so, hence the rather definite name.